3rdparty/gtest/gtest.h

```diff
@@ -10,4 +10,26 @@ inline std::string CurrentTestName() {
   return testing::GetUnitTestImpl()->current_test_info()->name();
 }
 
+// Effectively returns `argv[0]`.
```

Comment:
Then move `gtest-main.h` and `gtest-main-with-dflags.h` to `Bricks/gtest` => massive replace. Maybe not now?